src/cli/project/version/bump.rs

```diff
@@ -32,3 +72,71 @@ pub async fn execute(mut project: Project, bump_type: VersionBumpType) -> miette
 
     Ok(())
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use rattler_conda_types::Version;
+    use std::str::FromStr;
+
+    fn bump_version(version: &str, bump_type: VersionBumpType) -> String {
```

Comment:
This function is not used in the code itself? The contents of this code is also duplicated. Should this be moved out of the `test` module and used in the code above?

I think this code should also be moved to the [`Version::bump` code in `rattler`](https://github.com/conda/rattler/blob/32eefc87ef0f1bc5bcc0bb65183b97e71808f54c/crates/rattler_conda_types/src/version/bump.rs#L104).